tests/test_tree.py

```diff
@@ -122,14 +122,34 @@ def test_best_split_with_combination():
         tree.vectorised_array,
         tree.observed
     )
-
     assert list_ordered_equal(ndarr, orig_ndarr), 'Calling chaid should have no side affects for original numpy arrays'
     assert list_ordered_equal(arr, orig_arr), 'Calling chaid should have no side affects for original numpy arrays'
     assert split.column_id == 0, 'Identifies correct column to split on'
-    assert list_unordered_equal(split.split_map, [[1], [2, 3]]), 'Correctly identifies categories'
+    assert list_unordered_equal(split.split_map, [[1], [2], [3]]), 'Correctly identifies categories'
```

Comment:
Are we sure?

Reply:
Again, same here with the small base sizes.